e2e_test/batch/types/list/list_cast.slt.part

```diff
@@ -0,0 +1,61 @@
+statement ok
+SET RW_IMPLICIT_FLUSH TO true;
+
+query I
+select {1,2,3}::double[];
```

Comment:
FYI this is invalid in PostgreSQL and it requires `'{1,2,3}::double precision[]'`. But this can be considered a separate issue from this PR.